ASUWebPlatforms/webspark-ci
# WS2-1985: fix button problems

### Description
The problem was that the upcasting was not accounting for the sizes, so they were being added as separate elements afterwards. Adding `md|sm` to the possible classes fixed the issue. Also, while I was in there, I fixed the problem where there was white text on a white background.

QA Steps:
This is a little hard to test. The best way would be to view the multidev that Thomas created for the brandguide website. I pushed this fix to that site, which is where the problem was originally discovered, and now it is fixed.

To view the before and after difference of this change:

[On the live (broken) version of the brandguide site](https://live-brand-guide-d9-stable.ws.asu.edu/node/422/layout), do the following:
- Log in as an administrator
- Click on "Layout"
- Look in the "Creative and development vendors" section and see that the medium buttons are broken.
- Click the pencil icon for the "Creative and development vendors" section, and click "Configure."
- In the WYSIWYG editor, note that the buttons are broken.
- Click into the button text. It should display white text against the white background.

[On the ws2-1985 (fixed) multidev of the brandguide site](https://ws2-1985-brand-guide-d9-stable.ws.asu.edu/node/422/layout), do the following:
- Log in as an administrator
- Click on "Layout"
- Look in the "Creative and development vendors" section and see if the medium buttons display correctly.
- Click the pencil icon for the "Creative and development vendors" section, and click "Configure."
- In the WYSIWYG editor, check if the buttons render correctly.
- Click into the button text. It should turn the text black against the white background.
- If all of these things are true, it passes.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1985)

### Checklist


### Verified in browsers 


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
